CFC_DataCollector/recommender/augment_trips_pipeline.py

```diff
@@ -0,0 +1,26 @@
+import get_trips
+#import Profiles
+
+def augment_trips(trip_list):
```

Comment:
Should the functions be implemented here, or should they be implemented in a separate file and invoked from here. Having a separate file would make it easier to swap out implementations and re-arrange the pipelines.

ie. TripAlternatesGenerator.generateAlternates()